src/components/app/DragAndDrop.css

```diff
@@ -0,0 +1,36 @@
+.dragAndDropArea {
+  isolation: isolate;
+}
+
+.dropMessageWrapper {
```

Comment:
It'd be good to make these classes consistent in naming to avoid future naming conflicts. We usually use the component name to make up the names, so:

```
.dragAndDropMessageWrapper {}
.dragAndDropMessageWrapper.dragging {}
.dragAndDropMessage {}
```